package.json

```diff
@@ -62,11 +58,28 @@
     "@types/tmi.js": "^1.8.2",
     "@types/twitch-ext": "^1.24.4",
     "editorconfig-checker": "^5.0.1",
-    "eslint": "^8.33.0",
+    "eslint": "^8.42.0",
+    "eslint-config-prettier": "^8.8.0",
     "eslint-config-react-app": "^7.0.1",
```

Comment:
It looks like your config is no longer using this, so can we remove this dep if we don't need it?

Reply:
I think keeping `eslint-prettier-config` and `"extends": […, "prettier"]` is good practice to prevent eslint and prettier fighting or doing the same work twice. i would just keep it in there.

Reply:
:eyes: This comment was on the `eslint-config-react-app` line, not `eslint-config-prettier`

Reply:
oops, yeah that can probably go

Reply:
ah got it, my mistake 😅